src/pages/contributing/code.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Code
+description:
+  Code contributions can include anything from squashing bugs to adding feature
+  requests. Below you'll find everything you need to find a project and start
+  helping out.
+---
+
+export const Title = () => <span>Contribute code</span>;
+
+<PageDescription>
+
+Code contributions can include anything from squashing bugs to adding feature
+requests. Check out the instructions below to contribute code effectively.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Step 1: Find a project to work on</AnchorLink>
+<AnchorLink>Step 2: Set up your developer environment</AnchorLink>
+<AnchorLink>Step 3: Refine the issue</AnchorLink>
+<AnchorLink>Step 4: Iterate and get feedback</AnchorLink>
+
+</AnchorLinks>
+
+## Step 1: Find a project to work on
+
+New issues submitted by the community are initially triaged by Carbon team
+members. Any issue that the Carbon team accepts, but can not fit in their
+roadmap, is open for community contribution.
+
+#### Volunteer for existing work
+
+The best way to volunteer is to look through
+[existing GitHub issues](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22)
+labeled with `community contribution`. Any issue labeled with
+`code contribution needed` is fair game! Put a comment in the issue saying you'd
+like to help.
+
+If you want to join an issue that is already assigned to someone or has a
+specific milestone, please discuss with the assignee of the issue or with the
+Carbon team to coordinate.
+
+Issues labeled `good first issue` are great candidates to pick up if you are in
+the code for the first time. The Carbon team is also happy to help you find a
+get started. Just stop by our [office hours](/whats-happening/meetups/) or send
+us a slack message.
```

Comment:
Should we add the links to our slack channel here?

Reply:
Good idea! I'll add those.